spec/jobs/terms_of_use/sign_up_service_updater_job_spec.rb

```diff
@@ -16,6 +17,27 @@
       allow(MobileApplicationPlatform::SignUp::Service).to receive(:new).and_return(service_instance)
     end
 
+    it 'retries 15 times after failure' do
+      expect(described_class.get_sidekiq_options['retry']).to eq(15)
+    end
+
+    it 'logs a message when retries have been exhausted' do
+      logger_spy = instance_spy(ActiveSupport::Logger)
+      allow(Rails).to receive(:logger).and_return(logger_spy)
+      job_info = {
+        class: described_class.to_s,
+        args: %w[foo bar],
+        error_message: 'An error occurred'
+      }
+
+      described_class.sidekiq_retries_exhausted_block.call(job_info, nil)
```

Comment:
You should read the comment in `app/workers/evss/disability_compensation_form/submit_form526_cleanup.rb`

Sure showed them!

Reply:
"This can't be tested" is such a bold claim!

I looked at their `job_exhausted` method and they also [increment a Datadog metric](https://github.com/department-of-veterans-affairs/vets-api/blob/df80528e8f1d650f25a505952df05b56d2db1f69/lib/sidekiq/form526_job_status_tracker/job_tracker.rb#L67). Should we be doing something similar, or are we ok with just the log?

Reply:
Per @joeniquette, adding a metric here is unnecessary since he already has a dashboard in place to monitor exhausted jobs.